src/Presentation/Presenters/DonationConfirmationHtmlPresenter.php

```diff
@@ -29,25 +30,26 @@ public function __construct( TwigTemplate $template, UrlGenerator $urlGenerator,
 		$this->validation = $validation;
 	}
 
-	public function present( Donation $donation, string $updateToken, string $accessToken,
+	public function present( Donation $donation, array $paymentData, string $updateToken, string $accessToken,
 							 array $urlEndpoints ): string {
 		return $this->template->render(
-			$this->getConfirmationPageArguments( $donation, $updateToken, $accessToken, $urlEndpoints )
+			$this->getConfirmationPageArguments( $donation, $paymentData, $updateToken, $accessToken, $urlEndpoints )
 		);
 	}
 
-	private function getConfirmationPageArguments( Donation $donation, string $updateToken, string $accessToken,
+	private function getConfirmationPageArguments( Donation $donation, array $paymentData, string $updateToken, string $accessToken,
 		array $urlEndpoints ): array {
 		$donorDataFormatter = new DonorDataFormatter();
 		return [
 			'donation' => [
 				'id' => $donation->getId(),
-				'amount' => $donation->getAmount()->getEuroFloat(),
-				'interval' => $donation->getPaymentIntervalInMonths(),
-				'paymentType' => $donation->getPaymentMethodId(),
+				// TODO: Adapt the front end to take cents here for currency localisation
+				'amount' => Euro::newFromCents( $paymentData['amount'] )->getEuroFloat(),
```

Comment:
if you add a `amountInCents` field, we won't be under pressure to merge and deploy 2 repos (frontend and backend) at the same time. We can change the frontend and later deprecate/remove `amount` on the backend. Then we could do a 2nd round in the frontend, using "amount" with "amountInCents" as fallback, then renaming the field in the backend.